Formula/v8@5.8.rb

```diff
@@ -0,0 +1,118 @@
+# Track Chrome stable.
+# https://omahaproxy.appspot.com/
+class V8AT58 < Formula
+  desc "Google's JavaScript engine"
+  homepage "https://github.com/v8/v8/wiki"
+  url "https://chromium.googlesource.com/chromium/tools/depot_tools.git"
```

Comment:
Thanks for the `sync_gateway` suggestion, I see how do they handle caching, that's what I should also do. Though, I'm not sure about how should I handle `url`, as it would be misleading to clone v8 (and a waste of time as it huge) and not using it later.